### ISSUE TITLE
Question about the loss between student output and ae output

### ISSUE THREAD

**Issue by Youskrpig** [nelson1425/EfficientAD#11]
Hi， I'm puzzled about the  loss between student output and ae output，is it necessary?

**Comment by nelson1425** [comment]
Hi, yes it is necessary in my experience. In the paper the author says "The student learns the systematic reconstruction errors of the autoencoder on normal images, e.g., blurry reconstructions. At the same time, it does not learn the reconstruction errors for anomalies because these are not part of the training set. This makes the difference between the autoencoder’s output and the student’s output well-suited for computing the anomaly map."
